source/code/plugins/filter_scom_cor_match.rb

```diff
@@ -1,5 +1,7 @@
+require_relative 'scom_common'
+
 module Fluent
-  class SCOMCorrelatedMatchFilter < Filter
+  class SCOMCorrelatedMatchFilter < SCOMTimerFilterPlugin
```

Comment:
I don't really speak Ruby, so I'll leave this (and other Ruby changes) to Laura. I don't care much about commits, but if Laura wants it in a separate commit, you should do that (it is her project). I work a lot in mesos these days, and I get to live by their rules! 😃